README.MD

```diff
@@ -59,23 +59,33 @@ To see how to use the tool.
 yarn hardhat account create --private-key <0xstatic-private-key-of-the-cm-account-wallet> --network columbus
 ```
 The command will output your new **CM Account Address** in format 0x<\...\>. Store it and add it to your `.yml` configuration under `cm_account_addess`.
-To retrieve your bot wallet static private key, follow the instructions [here](https://docs.camino.network/guides/how-to-login-to-your-wallet#how-to-retrieve-your-private-key).  
+To retrieve your wallet static private key, follow the instructions [here](https://docs.camino.network/guides/how-to-login-to-your-wallet#how-to-retrieve-your-private-key).  
 
 2. To **add the bot** to the CM Account run:
 ```
 yarn hardhat account add:bot --bot <0xc-chain-address-of-the-bot-wallet> --cm-account  <0xCMAccount-address> --private-key <0xstatic-private-key-of-the-cm-account-wallet> --network columbus
 ```
-Add this bot's private key to your `.yml` configuration under `bot_key`.
+Add this **bot's wallet static private key** to your `.yml` configuration under `bot_key`.
 
-3. To **register a service** supported by your CM Account e.g.
+3. Granting Withdrawer role  
+
+In case you want to withdraw funds from the Cm Account wallet to another wallet, you have to grant it the `WITHDRAWER_ROLE`. To do this run:
```

Comment:
as well as same wallet.

```suggestion
In case you want to withdraw funds from the Cm Account wallet to a wallet, you have to grant it the `WITHDRAWER_ROLE`. To do this run:
```